### PR TITLE
[TRAFODION-3220] Executor GUI

### PR DESCRIPTION
**Pull request by zellerh**
Adding the ability to display query execution with the "display"
command in Trafodion (sqlci only, requires an X Windows display,
just as the compiler GUI does).

### CODE REVIEW

File: core/sql/SqlCompilerDebugger/ExeQueueDisplay.h [hunk 1]
```diff
@@ -33,22 +18,43 @@
 // under the License.
 //
 // @@@ END COPYRIGHT @@@
- *
- *
- *****************************************************************************
- */
+#ifndef EXEQUEUEDISPLAY_H
```
Comment (zellerh) [inline core/sql/SqlCompilerDebugger/ExeQueueDisplay.h]: I have no clue why git says I renamed ComSqlexedbg.h to ExeQueueDisplay.h. In reality, I deleted file ComSqlexedbg.h and added a new file ExeQueueDisplay.h. Please ignore the the ComSqlexedbg.h file, it doesn't have any meaningful content.

**Comment by sureshsubbiah** [comment]
+1. This is a wonderful change Hans. Can't wait to try it out.

**Comment by selvaganesang** [comment]
Is this feature available in debug mode build only?

**Comment by zellerh** [comment]
@selvaganesang, yes, the executor GUI is available in debug builds only, for performance reasons. The compiler GUI should work for both debug and release builds. This is controlled by the executor Makefile.

**Pull request merged**